app/uk/gov/hmrc/agentclientrelationships/repository/DeleteRecordRepository.scala

```diff
@@ -82,8 +81,6 @@ object DeleteRecord {
       (JsPath \ "gaToken").readNullable[String]
   )((a, s, t, g) => HeaderCarrier(authorization = a, sessionId = s, token = t, gaToken = g))
 
-  import ReactiveMongoFormats._
-
   implicit val formats: Format[DeleteRecord] = format[DeleteRecord]
```

Comment:
It doesn't look like this `format[DeleteRecord]` would serialise a Joda `DateTime` to the same as it would serialise a Java `ZonedDateTime`.
e.g. running the following in a scala console..
```
import play.api.libs.json.{Format, Json}

case class JodaDateTime(d: Option[org.joda.time.DateTime] = None)
case class JavaZonedDateTime(d: Option[java.time.ZonedDateTime] = None)

implicit val jodaDateTimeFormats: Format[JodaDateTime] = Json.format
implicit val javaZonedDateTimeFormats: Format[JavaZonedDateTime] = Json.format

Json.toJson(JodaDateTime(Some(org.joda.time.DateTime.now())))
Json.toJson(JavaZonedDateTime(Some(java.time.ZonedDateTime.now())))
```
results in:
```
res0: play.api.libs.json.JsValue = {"d":1560171805117}
res1: play.api.libs.json.JsValue = {"d":"2019-06-10T14:03:25.438+01:00[Europe/London]"}
```